source/2019-08-16-the-ember-times-issue-111.md

```diff
@@ -19,13 +20,18 @@ and last, but not least,
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Settle Down; Ember's Test Framework Has Your Async Covered 🕰️](https://www.linkedin.com/pulse/settle-down-embers-test-framework-has-your-async-covered-calvert/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+In the words of the author, "[t]he Ember community has spent a lot of time thinking about how to make your tests easy to write, dependable, and most importantly, **async aware**." It's easy in Ember to ensure that your synchronous tests wait for asynchronous actions to complete. 💎
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[@scalvert](https://github.com/scalvert) gives a great overview of how the [`settled`](https://github.com/emberjs/ember-test-helpers/blob/master/addon-test-support/@ember/test-helpers/settled.ts) helper plays a pivotal role. Thanks to `settled`, the test helpers that we use every day⁠—`render`, `click`, `visit`, and `triggerEvent`⁠—know when async operations have completed. We can also use `settled` directly to complete an async operation that [`@ember/test-helpers`](https://github.com/emberjs/ember-test-helpers) doesn't handle.
+
+The author also covers 2 additional tools that help us write async-aware tests:
+
+- [ember-test-waiters](https://github.com/rwjblue/ember-test-waiters), to mark the start and end of an async operation
```

Comment:
```suggestion
- [ember-test-waiters](https://github.com/rwjblue/ember-test-waiters), to mark the start and end of an async operation in order to integrate unmanaged async behaviors
```

Reply:
I think I get what you were thinking. I wanted to keep the sentence short and not include the technical term, deterministic.

I'll see if I can phrase this better, thanks.